docs/attributes-registry/android.md

```diff
@@ -13,17 +13,27 @@
 
 The Android platform on which the Android application is running.
 
-| Attribute              | Type   | Description                                                                                                                                                                                                                               | Examples   | Stability                                                        |
-| ---------------------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | ---------------------------------------------------------------- |
-| `android.os.api_level` | string | Uniquely identifies the framework API revision offered by a version (`os.version`) of the android operating system. More information can be found [here](https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels). | `33`; `32` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute | Type | Description | Examples | Stability |
+| --------- | ---- | ----------- | -------- | --------- |
+
+### `android.os.api_level`
```

Comment:
I believe this is not a good option, as it makes the pages long. 

I tried and I think it should work if we use simple HTML anchors. For example

```
<a id="android-os.api_level" href="#android-os.api_level">`android.os.api_level`</a>
```

Reply:
This should make both the cell text clickable and add a anchor to the id defined to the URL.

Reply:
yes @joaopgrassi I looked into that and noticed the difference too. Let me use the HTML anchors and see, or are you working on a commit already?

Reply:
Thanks for the feedback @joaopgrassi just tired it and it worked while retaining the table's structure.

Kindly review recent change and let me know how it looks